packages/cli/src/models/network/NetworkAppController.js

```diff
@@ -93,15 +93,25 @@ export default class NetworkAppController extends NetworkBaseController {
     log.error(`Possible initialization method 'initialize' found in contract. Make sure you initialize your instance.`);
   }
 
-  async upgradeProxies(packageName, contractAlias, proxyAddress, initMethod, initArgs) {
-    // Fetch all proxies that match the filters provided
-    const proxies = this.networkFile.getProxies({ package: packageName, contract: contractAlias, address: proxyAddress})
-    if (_.isEmpty(proxies)) {
-      log.info('No proxies to update were found');
-      return [];
-    }
+  async setProxiesAdmin(packageName, contractAlias, proxyAddress, newAdmin) {
+    const proxies = this._fetchOwnedProxies(packageName, contractAlias, proxyAddress)
+    if (proxies.length === 0) return [];
+    await this.fetch();
 
-    // Load project
+    await allPromisesOrError(
+      _.map(proxies, async (proxy) => {
+        log.info(`Changing admin of proxy ${proxy.address} to ${newAdmin}`)
+        await this.project.changeProxyAdmin(proxy.address, newAdmin)
+        this.networkFile.updateProxy(proxy, proxy => ({ ... proxy, admin: newAdmin }))
```

Comment:
shouldn't we set this value also when creating a new instance?

Reply:
We can safely assume that by default the admin of a new instance is the app